File: FrontendRazorPage/Models/AdminModel/VocabularyAdminModel.cs
using Microsoft.AspNetCore.Mvc.ModelBinding.Validation;
using System.Text.Json.Serialization;
namespace FrontendRazorPage.Models.AdminModel
{
    public class VocabularyAdminModel
    {
        public int Id { get; set; }
        public int LessonId { get; set; }
     
        public string ?Kanji { get; set; }
        public string Hiragana { get; set; }
        public string Romaji { get; set; }
        public string Meaning { get; set; }
    
        public string ? ExampleSentence { get; set; }
        public string? AudioUrl { get; set; }
        public DateTime CreatedAt { get; set; }
        public DateTime UpdatedAt { get; set; }
    }
}

File: FrontendAngular/src/app/Models/AdminModel/vocabulary-admin-model.ts
export interface VocabularyAdminModel {
    id: number;
    lessonId: number;
    kanji?: string;
    hiragana: string;
    romaji: string;
    meaning: string;
    exampleSentence?: string;
    textToSpeak?: string;

    // Các trường đặc thù của Admin để theo dõi dữ liệu
    createdAt?: string;
    updatedAt?: string;
}